Cargo.toml

```
@@ -1,7 +1,13 @@
+[workspace.package]
+edition = "2021"
+
 [workspace]
 members = ["crates/brioche", "crates/brioche-core", "crates/brioche-pack", "crates/brioche-test-support"]
 resolver = "2"
 
+[workspace.lints.clippy]
+all = { level = "deny", priority = -1 }
+
```

I'm pretty strongly against making warnings into hard errors by default, sometimes when I'm iterating I'll end up with unused variables and such and I'd prefer to be able to handle those as a separate cleanup step. So I'd prefer to get this reduced down to warn-by-default at least

```suggestion
[workspace.lints.clippy]
all = { level = "warn", priority = -1 }

```

...but definitely in favor of getting any potential warnings flagged as part of CI/CD, meaning any issues still need to be resolved (or allowed/silenced explicitly) before a PR could be merged. I think that _should_ already be covered by our CI/CD pipeline today, although looks like we should run checks on macOS as well considering all the warnings...

---

Sure, I see your point. I updated it. Thanks.